### ISSUE TITLE
Server side pagination

### ISSUE THREAD

**Issue by jevgeniP** [hjalmers/angular-generic-table#123]
Is it possible implement pagination from server side?

**Comment by hjalmers** [comment]
Yes, have you looked at the lazy loading example [here](https://hjalmers.github.io/angular-generic-table/lazy)?
